pkg/dt/node.go

```diff
@@ -120,6 +142,19 @@ func (n *Node) LookProperty(name string) (*Property, bool) {
 	return nil, false
 }
 
+// RemoveSubTreeAtIndex deletes the child at the given index (and all
+// its children).
+func (n *Node) RemoveSubTreeAtIndex(idx int) error {
+	if idx < 0 || idx >= len(n.Children) {
+		return fmt.Errorf("idx %d is not in range 0..%d: %w",
```

Comment:
I know if seems odd, but could you not break this line here.  I know in google it is common to break this way, but I would like to avoid that as much as we can. Otherwise this is fine. Thanks!

Reply:
ok, I might indeed have done that in more than one place. fixed here, thanks!

Reply:
These kinds of line breaks are not common at google either, we actually avoid them: https://google.github.io/styleguide/go/decisions#indentation-confusion